pkgs/development/compilers/computecpp/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, fetchzip, pkg-config, installShellFiles, ncurses5, ocl-icd, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "computecpp";
+  version = "1.2.0";
+
+  src = fetchzip {
+    url = "https://computecpp.codeplay.com/downloads/computecpp-ce/${version}/ubuntu-16.04-64bit.tar.gz";
+    sha256 = "191kwvzxfg1sbaq6aw6f84chi7bhsibb2a63zsyz3gz8m0c0syr5";
+    stripRoot = true;
+  };
+
+  dontStrip = true;
+
+  nativeBuildInputs = [ pkg-config installShellFiles ];
+
+  patchPhase = let
+    binaries = [
+      # Not `compute` and `compute-cl` -- these are symlinks.
+      "compute++"
+      "computecpp_info"
+      "ld.ldd"
+      "ldd"
+      "ldd-link"
+      "llvm-spirv"
+      "module-pack-wrapper"
+      "sycl-bundler"
+    ];
+    libaries = stdenv.lib.makeLibraryPath [
+      stdenv.cc.cc.lib
+      ncurses5
```

Comment:
I'd appreciate advice from reviewers on how best to fix a issue with this package. When running the binary, I get the following output:

```
/nix/store/anf6w9ad9577d121w8v2jxhp9m09ycil-computecpp-1.2.0/bin/compute: /nix/store/7fnhs9lg64fgzx13d2skjrckiic2yqsk-ncurses-6.1-20190112-abi5-compat/lib/libtinfo.so.5: no version information available (required by /nix/store/anf6w9ad9577d121w8v2jxhp9m09ycil-computecpp-1.2.0/bin/compute)
```

I believe this is due to the original binary being compiled with an older version of `libtinfo.so.5`, but I'm not sure what I can do about that.

Reply:
My guess is that `libtinfo.so` has been stripped of debug symbols. Does this prevent to program from starting?

Reply:
It doesn't, it appears to work fine but just prints that message when invoked.